lib/v2/datasource/remote/api/rates_repository.dart

```diff
@@ -16,4 +16,32 @@ class RatesRepository extends NetworkRepository {
             }))
         .catchError((error) => mapHttpError(error));
   }
+
+  Future<Result> getFiatRatesAlternate() async {
+    print("[http] get alternate fiat rates");
+
+    return http
+        .get("http://data.fixer.io/api/latest?access_key=${Config.fixerApiKey}&symbols=CRC,GTQ,USD")
+        .then((http.Response response) => mapHttpResponse(response, (dynamic body) {
+              return FiatRateModel.fromJsonFixer(body);
+            }))
+        .catchError((error) => mapHttpError(error));
+  }
+
+  Future<Result> getUSDRate() async {
+    print('[http] get seeds rate USD');
+
+    final rateURL = '$baseURL/v1/chain/get_table_rows';
```

Comment:
same with this

Reply:
Yes, you are right, this var is not need i can put direct the url in the htttp method.